dags/weekly_nps_pilotage.py

```diff
@@ -0,0 +1,64 @@
+from airflow import DAG
+from airflow.decorators import task
+from airflow.operators import empty
+
+from dags.common import dataframes, dates, db, default_dag_args, slack
+
+
+with DAG(
+    "nps_hebdo_pilotage",
+    schedule_interval="@weekly",
+    **default_dag_args(),
+) as dag:
+    start = empty.EmptyOperator(task_id="start")
+
+    end = slack.success_notifying_task()
+
+    @task(task_id="create_weekly_nps")
+    def create_weekly_nps(**kwargs):
+        import pandas as pd
+
+        week_list = dates.week_list()
+        df = db.create_df_from_db('SELECT * FROM "suivi_satisfaction"')
+
+        tb_pe = [
+            "tb 169 - Taux de transformation PE",
+            "tb 162 - Fiches de poste en tension PE",
+            "tb 149 - Candidatures orientées PE",
+            "tb 168 - Délai d'entrée en IAE",
+        ]
+
+        tb_siae = [
+            "tb 165 - Recrutement SIAE",
+        ]
+
+        df_nps = []
+        df_nps_pe = []
+        df_nps_siae = []
+
+        for i in week_list:
+            # NPS de tous les TBs
+            df_cur_week = df[df["Date"] <= i]
+            df_nps.append(dataframes.get_nps(df_cur_week, i))
+            # NPS TB Pôle emploi
+            df_cur_week_pe = df_cur_week[df_cur_week["Nom Du Tb"].isin(tb_pe)]
+            df_nps_pe.append(dataframes.get_nps(df_cur_week_pe, i))
+            # NPS TB SIAE
+            df_cur_week_siae = df_cur_week[df_cur_week["Nom Du Tb"].isin(tb_siae)]
+            df_nps_siae.append(dataframes.get_nps(df_cur_week_siae, i))
+
+        df_nps = pd.concat(df_nps)
+        df_nps_pe = pd.concat(df_nps_pe)
+        df_nps_siae = pd.concat(df_nps_siae)
+
+        df_nps["tb"] = "Tous les TBs"
+        df_nps_pe["tb"] = "TBs PE"
+        df_nps_siae["tb"] = "TBs SIAE"
+
+        df = pd.concat([df_nps, df_nps_pe, df_nps_siae])
```

Comment:
ça me semble bizarre de prendre chaque sous tableau pour les repartir en 3 tableaux, puis modifier les 3 tableaux et le reconcaténer en 1. On ne peut pas juste itérer sur les lignes et remplacer le nom du tb dans chaque cas ?